tuba/define_geometry.py

```diff
@@ -406,9 +406,9 @@ def Bent(bending_radius,arg1="",arg2="",arg3="intersect",name=""):
 
    #TO DO: when intersection problem arises (no vector before bent to move old end_tubapoint)--> switch automatically to add and bring a warning
 
-    if arg3=="intersect":
+    if arg3=="intersect" and arg1!="":
         tub.current_tubapoint.pos=tub.current_tubapoint.pos - \
-                                     tub.current_tubapoint.vd2x*bending_radius
+                                     tub.current_tubapoint.vd2x*bending_radius*math.sin(arg1 * math.pi / 180)
```

Comment:
Sorry, I was wrong here. It shouldn't be sin(alpha) but rather tan(alpha/2) - please close this pull request.

Reply:
thank you very much already for the contribution. have overseen this bug.  i'm about to test it - anyway i think to rewrite the function to make it clearer and more explicit (its a bit convoluting).